src/pages/Dashboard/components/UserInformation/UserInformation.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, useTheme, /* styled, Paper, */ Grid, useMediaQuery, Typography } from '@mui/material'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import React from 'react'
+
+
+
+const UserInformation = (): JSX.Element => {
+
+    const theme = useTheme()
+    /* const Item = styled(Paper)({
+        padding: theme.spacing(1),
+        textAlign: 'center',
+        color: theme.palette.text.secondary,
+    }); */
+
+    const isMd = useMediaQuery(theme.breakpoints.up('md'));
+    return (
+        <Box sx={{ m: 2 }}>
+            User information / Number of appointments left.
```

Comment:
Can you initiate translations???  you need to use the:

const {t} = useTranslation()

please check how I did it in the other components. This is very important to learn!

For spanish:  Informacion de usuario